src/create_env.rs

```diff
@@ -0,0 +1,49 @@
+use std::time::Duration;
+
+use anyhow::Error;
+use btc_rpc_proxy::{AuthSource, Env, Peers, RpcClient, TorEnv, Users};
+use slog::Drain;
+use tokio::sync::Mutex;
+
+use config::ResultExt;
+
+include_config!();
```

Comment:
This macro is currently screwed up, we need to expand it manually for the time being:

```rust
mod config {
    include!(concat!(env!("OUT_DIR"), "/configure_me_config.rs"));
}
use self::config::{Config, ResultExt};
```